src/vk/Generated/Unions.gen.cs

```diff
@@ -5,35 +5,6 @@
 namespace Vulkan
 {
     [StructLayout(LayoutKind.Explicit)]
-    public struct VkClearColorValue
```

Comment:
I'd prefer to just keep this being auto-generated like it was before. We can still add the helper constructors, though. I think those are a good idea.